### PR TITLE
Add servicemonitor

### PR DESCRIPTION
**Pull request by muffl0n**
See #361

### CODE REVIEW

File: deploy/charts/external-secrets/templates/webhook-service.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
   annotations:
     prometheus.io/path: "/metrics"
     prometheus.io/scrape: "true"
-    prometheus.io/port: {{ .Values.prometheus.service.port | quote }}
```
Comment (moolen) [inline deploy/charts/external-secrets/templates/webhook-service.yaml:12]: Can we remove all these annotations and instead use the ServiceMonitor from above to target this service aswell?

**Review by moolen — COMMENTED** [review]
Hey, thank you for your contribution! I'm fine to introduce these breaking changes. Can we maybe get rid of the annotations of the other service (see comment above)?

**Comment by muffl0n** [comment]
Absolutely! Like mentioned in #361 I saw that those were just recently added. I was hoping that I could team up with @gusfcarvalho to change those, as I do not have the chance to test them, atm. Also I wanted to make sure they have a chance to get merged before I take the time. :)

**Comment by gusfcarvalho** [comment]
hey @muffl0n ! I think these changes would roll out in the same manner for the webhook and the certController. You can actually test them locally as well by using kind or minikube (although with minikube you'll need to do a eval $(minikube docker-env) before):

```
Assuming you are on linux:
make docker.build BUILD_ARGS="--build-arg TARGETARCH=amd64 --build-arg TARGETOS=linux"
make generate
make helm.generate
# replace with the version outputted by make docker build
helm install external-secrets deploy/charts/external-secrets/ --set image.tag=$version --set webhook.image.tag=$version --set certController.image.tag=$version
```

**Comment by gusfcarvalho** [comment]
Hi @muffl0n ! Any updates on this PR?

**Comment by muffl0n** [comment]
Unfortunately not. Cause of two launches (one at my job and one personal ;)), I can't currently find the time to work on this.

**Comment by hanzala1234** [comment]
Is there any update on that? looking forward for this PR to be merged!

**Comment by muffl0n** [comment]
Thank you! I really appreciate it! 
Sorry for letting you down, but things are a bit stressful at the moment. :(

**Comment by moolen** [comment]
No worries, all good! I've opened #917 for adding ServiceMonitor.

**Pull request closed without merging**